controllers/validator.go

```diff
@@ -23,6 +23,10 @@ func (r *DPAReconciler) ValidateDataProtectionCR(log logr.Logger) (bool, error)
 		return false, errors.New("DPA CR Velero configuration cannot be nil")
 	}
 
+	if dpa.Spec.Configuration.Restic != nil && dpa.Spec.Configuration.NodeAgent != nil {
+		return false, errors.New("It's not allowed to have both 'restic' (deprecated in OADP 1.3) and 'nodeAgent' options simultaneously")
```

Comment:
++ on the validation, NIT: maybe just re-frame the error message: " DPA CR cannot have `restic` as well as  `nodeAgent` enabled at the same time/simultaneously"

Reply:
Changed to: 
"DPA CR cannot have restic (deprecated in OADP 1.3) as well as nodeAgent options at the same time"

I did not add "enable" because there is field `enabled` which may be misleading here, they can't be defined at all not only enabled: true.